string name
---
int32 status
string value
string type